flocker/control/_protocol.py

```diff
@@ -81,7 +88,8 @@ class ClusterStatusCommand(Command):
     in the convergence agent during startup.
     """
     arguments = [('configuration', DeploymentArgument()),
-                 ('state', DeploymentArgument())]
+                 ('state', DeploymentArgument()),
+                 ('eliot_context', Unicode())]
```

Comment:
A custom `IArgument` implementation for eliot actions probably makes sense.  This would let application code pass an action as an argument to the `callRemote` call and let the `IArgument` figure out how to serialize and unserialize the object.  This will be increasingly convenient as there are more AMP calls (with just two it's not so bad but we're going to have more, probably for result reporting and definitely for node-to-node communication).